standard/locale.lua

```diff
@@ -62,17 +62,17 @@ function Locale.formatLocations(args)
 		-- Always normalize region name
 		if not location.region and location.country then
 			-- Check if the country provided is actually a region
-			location.region = String.nilIfEmpty(Region.run{region = location.country, onlyRegion = true})
+			location.region = String.nilIfEmpty(Region.run{region = location.country, onlyRegion = true} --[[@as string]])
 
 			-- If it actually was a region, it's no longer a country, otherwise get the Region from the country
 			if location.region then
 				location.country = nil
 			else
-				location.region = String.nilIfEmpty(Region.run{country = location.country, onlyRegion = true})
+				location.region = String.nilIfEmpty(Region.run{country = location.country, onlyRegion = true} --[[@as string]])
 			end
 
 		elseif location.region then
-			location.region = String.nilIfEmpty(Region.run{region = location.region, onlyRegion = true})
+			location.region = String.nilIfEmpty(Region.run{region = location.region, onlyRegion = true} --[[@as string]])
```

Comment:
why's this needed?

Reply:
because region returns table|string

Reply:
planned for it to be removed again after adjusts for #3029